java-client/build.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
There are too many shell scripts in the project, and we are going to clean them up, it's not suggest to add more shell scripts now. You can describe the build/install/test commands in README instead.

Reply:
ok